doc/releases/release-notes-3.3.rst

```diff
@@ -355,13 +393,21 @@ Drivers and Sensors
 
 * ADC
 
+  * STM32: Now Supports sequencing multiple channels into a single read.
+
+* Battery-backed RAM
+
+  * STM32: Add driver to enable support for backup registers from RTC.
```

Comment:
Please use the past tense to be consistent with the rest.

```suggestion
  * STM32: Added driver to enable support for backup registers from RTC.
```

Reply:
Right, txs. Fixed